analyzer/restapi-plugin/src/main/java/eu/fasten/analyzer/restapiplugin/KnowledgeBaseConnector.java

```diff
@@ -195,25 +196,33 @@ public void setInitDepGraphRequest() {
      */
     @PostConstruct
     public void connectToGraphDB() {
-        logger.info("Establishing connection to the Graph Database at " + graphdbPath + "...");
-        try {
-            graphDao = RocksDBConnector.createReadOnlyRocksDBAccessObject(graphdbPath);
-        } catch (RuntimeException e) {
-            logger.error("Couldn't connect to the Graph Database", e);
-            System.exit(1);
+        switch (KnowledgeBaseConnector.forge) {
```

Comment:
Why using a `switch` statement here? You could use an if statement.

Reply:
I used the `switch` statement for consistency purposes, as all forge checks in the REST-API have been performed `switch` statement s, but I changed it to an `if` statement accordingly.